docker-compose.yml

```diff
@@ -157,15 +171,14 @@ services:
     volumes:
       - ./.git:/app/.git
     environment:
+      <<: [*rabbit-mq-user-and-pass, *mailjet-api-id-and-secret, *max-number-of-days-selectable-for-api-key-validity, *data-request-upload-max-file-size-in-megabytes]
```

Comment:
Ich glaube `*mailjet-api-id-and-secret` brauchen die e2e tests nicht.

Reply:
removet